File: 1470.shuffle-the-array/solution.kt
class Solution {
    fun shuffle(nums: IntArray, n: Int): IntArray {
        val results = mutableListOf<Int>()
        for (i in 0..n-1) {
            results.add(nums[i])
            results.add(nums[n + i])
        }
        return results.toIntArray()
    }
}

File: 1470.shuffle-the-array/solution.py
class Solution:
    def shuffle(self, nums: List[int], n: int) -> List[int]:
        p1 = 0
        p2 = n
        N2 = len(nums)
        results = []
        while (p2 < N2):
            results.append(nums[p1])
            results.append(nums[p2])
            p1 += 1
            p2 += 1
        return results
